packages/foam-vscode/src/features/open-daily-note.ts

```diff
@@ -69,7 +71,7 @@ async function createDailyNoteIfNotExists(
 
   await fs.promises.writeFile(
     dailyNotePath,
-    `# ${dateFormat(currentDate, titleFormat, false)}\r\n`
+    `# ${dateFormat(currentDate, titleFormat, false)}${docConfig.eol}${docConfig.eol}`
```

Comment:
I am assuming that `docConfig` would be initialized before this command is run. Is that a false assumption?

Reply:
you are correct. `docConfig` will be initialized when the file is imported, so you are guaranteed that here it will be valid